src/Symfony/Component/Config/Definition/Builder/ExprBuilder.php

```diff
@@ -21,14 +21,7 @@
  */
 class ExprBuilder
 {
-    public const TYPE_ANY = 'any';
-    public const TYPE_STRING = 'string';
-    public const TYPE_NULL = 'null';
-    public const TYPE_ARRAY = 'array';
-
     protected $node;
-
-    public $allowedTypes;
```

Comment:
removing consts and public properties is a BC break, we need to figure out a way that doesn't break BC